python/GafferImageUI/ImageReaderUI.py

```diff
@@ -270,6 +270,17 @@
 
 		],
 
+		"fileValid" : [
+			"description",
+			"""
+			Whether or not the files exists and can be read into memory,
+			value calculated per frame if an image sequence.
+			""",
```

Comment:
We should document the interaction with MissingFrameMode and FrameMaskMode here, once we've settled on the right behaviour.